.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: csharp
+
+os:
+ - linux
+ - osx
+
+dist: trusty
+osx_image: xcode9.1
+
+mono: none
+dotnet: 2.1.300
+
+before_install:
+  - cd tests/neo-vm.Tests
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ulimit -n 2048; fi
+
+script:
+ - dotnet restore
+ - dotnet test
```

Comment:
That's fine for me, I never done this configuration before. It will be very helpful!

Reply:
What configuration do you use?

Reply:
I mean, never configured travis before.